.github/workflows/rds-plan.yml

```diff
@@ -0,0 +1,40 @@
+name: rds plan terraform
+
+on:
+  pull_request:
+    paths:
+      - terraform/services/tfstate/rds/**
```

Comment:
Paths should be the service (terraform/services/api-rds/**) and the workflow (.github/workflows/api-rds-plan.yml)